irods/__init__.py

```diff
@@ -3,6 +3,19 @@
 import logging
 import os
 
+def env_filename_from_keyword_args(kwargs):
```

Comment:
The reason for the use of `kwargs` as a bare parameter for the `pop` is related to this:

```
map = {"a":1}
def as_keywords( **kwargs ): kwargs.pop('a') # this will not alter the passed-in dictionary (the "**" creates a copy)
def as_argument( kwargs): kwargs.pop('a')    # this will alter the passed-in dictionary
as_keywords(**map) ; print (f"as_keywords leaves map as {map}")
as_argument(map) ; print (f"as_arguments leaves map as {map}")
```
So if the `pop` method is to have an effect on the kwargs dict, as is proper for its use [in this function](https://github.com/d-w-moore/python-irodsclient/blob/faa6080f597ced583838fd4c697c3e6ece536446/irods/test/helpers.py#L169) , then kwargs needs to be passed as an argument, rather than rendered into a series of keyword options for the function call

Reply:
Ohhh, I see. We actually want to modify it, which is why it's not a kwarg. Okay, I just wanted to make sure that was intentional. Resolving.